### ISSUE TITLE
script runs dryrun first, only purges services on 2. run, wrong console display?

### ISSUE THREAD

**Issue by zoulztealer** [madbomb122/BlackViperScript#24]
**Describe the bug**
not sure if this is intended, but there are a few things i have noticed, that were not there in the versions before.

1. when clicked on run script, the script does a dryrun / list first even when there is no dry run option activated. only when run script is clicked another time after the first run, the script purges the services.

2. after having purged services, the console window shows unclear categories "msg1":

```
>  Changing Service Please wait...
> msg1Tweaked
>  Service_Name - Current -> Change_To
> ----------------------------------------
```

and

```
> ----------------------------------------
>  Service Changed...
> msg13
> msg1161
> msg13
> msg10
> msg100:00:01.7787463
```

**To Reproduce**
1. run script
2. check console window / script behaviour

**Expected behavior**
the script should not dry run when its not activated. run script should cause immediate purge of services. msg1 should display correct values.

**Screenshots**
![bvscript1](https://user-images.githubusercontent.com/42149920/44703824-5d7d2180-aa99-11e8-814e-34221f044ea5.png)
![bvscript2](https://user-images.githubusercontent.com/42149920/44703827-5fdf7b80-aa99-11e8-8da4-dc69e07efbba.png)
![bvscript3](https://user-images.githubusercontent.com/42149920/44703831-6110a880-aa99-11e8-8f3b-bcc5ee3ef0e5.png)
![bvscript4](https://user-images.githubusercontent.com/42149920/44703833-62da6c00-aa99-11e8-974c-5251ab48561e.png)
![bvscript5](https://user-images.githubusercontent.com/42149920/44703842-71288800-aa99-11e8-9d8c-c82a43d48b69.png)

**Desktop (please complete the following information):**
 - OS Version: win 10 pro 64
 - OS Build: 17134.228

**Additional context**
 - Script Version: 5.2.1 (aug 26 2018)
- service version: 4.0 (may 29 2018)
 - Release Type: stable

**Comment by madbomb122** [comment]
Thanks for letting me know,
I found the bug with that msg1 thing (simple mistake in script), it's supposed to display text instead of msg1 -Fix will be in next release

as for the dry-run.. that ran because of the dev log option. (I should put a note in there about that) -Note will be in next release.

As for the 2nd run not doing a dry run (when it should have with the dev log checked) is something i have to check into.

**Comment by madbomb122** [comment]
Issue should be Fixed and has been release (in v5.2.2)

If issue still happens please re-open, thanks
